cabal.project

```diff
@@ -11,7 +11,7 @@ repository cardano-haskell-packages
 
 -- The hackage index-state
 index-state: 2025-05-14T16:20:57Z
-index-state: cardano-haskell-packages 2025-05-14T07:44:24Z
+index-state: cardano-haskell-packages 2025-04-08T17:22:48Z
```

Comment:
This change was necessary in order to be able to reuse the flake inputs that do not cause a problem on MacOS.

Reply:
You should not be downgrading `index-state`

Reply:
Thanks to @angerman 🙌 he found that removing the dependency on `ghcid` also removes the dependency on `hfsvnt` from `Cocoa` on MacOS. Since we do not use `ghcid` any more, this unblocks us. 👍